eth-proof-extractor/index.js

```diff
@@ -43,13 +27,16 @@ class EthProofExtractor {
     return tree
   }
 
-  async extractProof(web3, block, tree, transactionIndex) {
+  async extractProof(block, tree, transactionIndex) {
     const [, , stack] = await promisfy(
       tree.findPath,
       tree
     )(encode(transactionIndex))
 
-    const blockData = await web3.eth.getBlock(block.number)
+    // is this necessary? how different are these?
+    //   * robustWeb3.getBlock (the passed-in block)
+    //   * robustWeb3.web3.eth.getBlock
```

Comment:
@ailisp do you know the answer to this question?

Reply:
web3.eth.getBlock sometimes failed and crashed because websocket disconnection. robustWeb3 is a wrapper to web3, handle websocket disconnection, reconnect and retry. We should always use robustWeb3 (some usage of web3 hasn't been replaced by robustWeb3, but that's the plan)